app/docker/views/networks/edit/network.html

```diff
@@ -41,9 +41,21 @@
               <td>Internal</td>
               <td>{{ network.Internal }}</td>
             </tr>
-            <tr ng-if="network.IPAM.Config.length > 0" ng-repeat="config in network.IPAM.Config">
-              <td>Subnet - {{ config.Subnet }}</td>
-              <td>Gateway - {{ config.Gateway }}</td>
+            <tr ng-if="network.IPAM.IPV4Config.length > 0" ng-repeat-start="config in network.IPAM.IPV4Config">
+              <td>IPV4 Subnet - {{ config.Subnet }}</td>
+              <td>IPV4 Gateway - {{ config.Gateway }}</td>
+            </tr>
+            <tr ng-if="network.IPAM.IPV4Config.length > 0" ng-repeat-end>
+              <td>IPV4 IP range - {{ config.IPRange }}</td>
+              <td>IPV4 Excluded Ips - {{ config.AuxAddress }}</td>
+            </tr>
```

Comment:
Using a single `ng-repeat` instead of using `ng-repeat-start` and `ng-repeat-end` should produce the same output, right ? If yes, change to:

```
<tr ng-if="network.IPAM.IPV4Config.length > 0" ng-repeat="config in network.IPAM.IPV4Config">
  <td>IPV4 Subnet - {{ config.Subnet }}</td>
  <td>IPV4 Gateway - {{ config.Gateway }}</td>
</tr>
<tr ng-if="network.IPAM.IPV4Config.length > 0">
  <td>IPV4 IP range - {{ config.IPRange }}</td>
  <td>IPV4 Excluded Ips - {{ config.AuxAddress }}</td>
</tr>
```

Reply:
I want to alternate between IPV4 Subnet | IPV4 Gateway and IPV4 IP range | IPV4 excluded ips in the table. I don't have the same output with ng-repeat.